### DISCUSSION TITLE
Amazon Lightsail External vs. Internal IP issues

### DISCUSSION THREAD

**Discussion by mdsilvers** [jamulussoftware/jamulus#1901] [Technical Support & Help]
Hello.  I've installed my server following the instructions WiKi for my server.  My issue is the Jamulus server is pulling my 'internal' IP address instead of my 'external' IP address and I can't connect to my private server.  I'm not sure if I need port forwarding, etc to get this to work?  See my info below for the server:

My external IP: 3.0.53.164

Server Status:

● jamulus-headless.service - Jamulus headless server
     Loaded: loaded (/lib/systemd/system/jamulus-headless.service; enabled; vendor preset: enabled)
     Active: active (running) since Wed 2021-07-07 07:08:15 UTC; 1h 33min ago
   Main PID: 396 (jamulus-headles)
      Tasks: 2 (limit: 555)
     Memory: 47.8M
     CGroup: /system.slice/jamulus-headless.service
             └─396 /usr/bin/jamulus-headless -s -n

Jul 07 07:08:15 ip-172-26-14-223 systemd[1]: Started Jamulus headless server.
Jul 07 07:08:16 ip-172-26-14-223 jamulus[396]: - server mode chosen
Jul 07 07:08:16 ip-172-26-14-223 jamulus[396]: - no GUI mode chosen
Jul 07 07:08:16 ip-172-26-14-223 jamulus[396]: Using "172.26.14.223" as external IP.
Jul 07 07:08:16 ip-172-26-14-223 jamulus[396]: Recording state not initialised
Jul 07 07:08:16 ip-172-26-14-223 jamulus[396]:  *** Jamulus, Version 3.8.0
Jul 07 07:08:16 ip-172-26-14-223 jamulus[396]:  *** Internet Jam Session Software
Jul 07 07:08:16 ip-172-26-14-223 jamulus[396]:  *** Released under the GNU General Public License

**Comment by hoffie** [comment]
Hi.

> My issue is the Jamulus server is pulling my 'internal' IP address instead of my 'external' IP address and I can't connect to my private server.

The log message is just informational and I don't think the information is used anywhere as long as you are not registering to a Directory server.

> I'm not sure if I need port forwarding, etc to get this to work? 

For a publicly reachable server, you will definitely need port forwarding in your scenario (unless you want to resort to publishing it to a public directory server), yes.

**Comment by mdsilvers** [comment]
Thanks Christian. The instructions for Amazon Lightsail did not mention port forwarding, so that is why I am confused.  I’ll try to figure it out.

 

From: Christian Hoffmann ***@***.***>
Reply-To: jamulussoftware/jamulus ***@***.***>
Date: Wednesday, July 7, 2021 at 4:48 PM
To: jamulussoftware/jamulus ***@***.***>
Cc: mdsilvers SILVERSTEIN ***@***.***>, Author ***@***.***>
Subject: Re: [jamulussoftware/jamulus] Amazon Lightsail External vs. Internal IP issues (#1901)

 

Hi.

My issue is the Jamulus server is pulling my 'internal' IP address instead of my 'external' IP address and I can't connect to my private server.

The log message is just informational and I don't think the information is used anywhere as long as you are not registering to a Directory server.

I'm not sure if I need port forwarding, etc to get this to work?

For a publicly reachable server, you will definitely need port forwarding in your scenario (unless you want to resort to publishing it to a public directory server), yes.

**Comment by mdsilvers** [comment]
I got it working!  Thanks!!

 

From: Christian Hoffmann ***@***.***>
Reply-To: jamulussoftware/jamulus ***@***.***>
Date: Wednesday, July 7, 2021 at 4:48 PM
To: jamulussoftware/jamulus ***@***.***>
Cc: mdsilvers SILVERSTEIN ***@***.***>, Author ***@***.***>
Subject: Re: [jamulussoftware/jamulus] Amazon Lightsail External vs. Internal IP issues (#1901)

 

Hi.

My issue is the Jamulus server is pulling my 'internal' IP address instead of my 'external' IP address and I can't connect to my private server.

The log message is just informational and I don't think the information is used anywhere as long as you are not registering to a Directory server.

I'm not sure if I need port forwarding, etc to get this to work?

For a publicly reachable server, you will definitely need port forwarding in your scenario (unless you want to resort to publishing it to a public directory server), yes.
